lib/interpreters/split_clustering.rb

```diff
@@ -1162,26 +1279,27 @@ def remove_used_empties_and_refills(vrp, result)
       def tree_leafs(graph, node)
         if node.nil?
           [nil]
-        elsif (graph[node][:level]).zero?
-           [node]
-         else
-           [tree_leafs(graph, graph[node][:left]), tree_leafs(graph, graph[node][:right])]
-         end
-      end
-
-      def tree_leafs_delete(graph, node)
-        returned = if node.nil?
-          []
         elsif (graph[node][:level]).zero?
           [node]
         else
           [tree_leafs(graph, graph[node][:left]), tree_leafs(graph, graph[node][:right])]
         end
+      end
+
+      def tree_leafs_delete(graph, node)
```

Comment:
_tree_leafs_delete_ and _tree_leafs_ seem very similar. Could we consider having only _tree_leafs_ function but with an option to delete ?

Reply:
I haven't modified `tree_leafs_delete` in purpose, it shouldn't be in my dev at all :confused: I am confused!

Reply:
Oh it is the syntax correction. I apparently corrected the `if` conditional assignment but the dev has nothing to do with `tree_leafs` functions. They are used in `split_hierarchical`. So I would rather not make such a change in this PR.

Reply:
In fact I could not see the difference :p maybe some spaces where removed ?

Reply:
If you check with "Hide whitespace changes" option of the review page, then the diff becomes more obvious and minimal.